exercises/practice/two-fer/.meta/example.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE twofer
+SET response = 'One for ' ||
```

Comment:
The rest of the code uses 4-space indent. This is the only place where a hanging indent is used.